napari/_vispy/utils/visual.py

```diff
@@ -58,7 +58,7 @@
 }
 
 
-overlay_to_visual = {
+overlay_to_visual: Dict[Type[Overlay], Type[VispyBaseOverlay]] = {
```

Comment:
Why is Type needed here?

Reply:
Because the keys and values are class types not instances of those classes.